modules/ROOT/pages/7.0-release-notes.adoc

```diff
@@ -217,6 +217,12 @@ For more information about `highlight_on_focus` see the xref:accessibility.adoc#
 [NOTE]
 Any editors using this `highlight_on_focus: true` option, can remove this option from their {productname} init configuration when upgrading to {productname} 7.0.
 
+=== Change table height resizing handling to remove heights from `td`/`th` elements and only apply to `tr` elements.
+
+Previously, when resizing a table using either the row resize bars or corner resize handles, any existing `height` styles are removed from the `td`/`th` elements and only apply to `tr` elements and table element. Additionally, the heights calculated and applied for the `tr` elements and table element more accurately represent the actual visual height in the editor.
```

Comment:
It seems like the previous behaviour doesn't remove any of the existing `height` styles from the `td`/`th` elements. both the cells and the rows (`tr`) would retain their set heights.

Reply:
@ltrouton Can you update the above?

Reply:
It's all good now since @ltrouton has removed "previously", thanks!